app/uk/gov/hmrc/helptosavereminder/controllers/test/TestController.scala

```diff
@@ -39,7 +39,7 @@ class TestController @Inject()(
       .map(_ => Ok)
   }
 
-  def getHtsUser(nino: String): Action[AnyContent] = Action.async { implicit request =>
+  def getHtsUser(nino: String): Action[AnyContent] = Action.async {
```

Comment:
The request wasn't being used here either. Will there be any consequences if the implicit request is removed if it's not referred to in that method?